string filenamebase
bool arenastate
bool video
bool bag
bool onlyWhileTriggered

